controlsfx/src/main/java/org/controlsfx/control/table/FilterPanel.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2015, 2019, ControlsFX
+ * Copyright (c) 2015, 2016, 2020 ControlsFX

```

Comment:
This should be updated to "Copyright (c) 2015, 2020, ControlsFX"

Reply:
fixed now.